docs/docs/contributors/developers-guide/adding-language.md

```diff
@@ -0,0 +1,69 @@
+# How to add a language
+
+1. A language must be added into [Crowdin](https://crowdin.com/project/mealie). This is typically done by hay-kot, and should be asked directly onto the [Discord server](https://discord.gg/QuStdQGSGK) in the #translation channel.
+2. After the language is added, an automatic PR is created and accepted. This can take some hours.
```

Comment:
```suggestion
2. After the language is added to Crowdin, a PR is automatically created, which will then be merged by one of the maintainers. It may take several hours for the PR to be created in GitHub.
```